### ISSUE TITLE
[wiki] Mod compatibility

### ISSUE THREAD

**Issue by nitrogenez** [amblelabs/ait#833]
# TL;DR

I suggest adding a mod compatibility entry to the wiki.

# The what

The mod compat entry must contain infoon compatible versions of other popular mods (e.g
 Sodium), as not all versions work properly/work at all with AIT. E.g, almost any version of Sodium works fine with AIT, but this changes if Immersive Portals is installed.

# The why

Some people might want to make modpacks with AIT, so it could be a great QoL addition for people, as they wouldn't need to manually check every version, as it would be already done for them.

# Why would we do that?

The question is, why do that, if the users can do it by themselves? Fair point, but I think that we should at least specify compatible versions of mods that we mention in the readme. It's a good practice to specify version of optional project dependencies.

# The process

- Install mod N of latest version alongside AIT
- If doesn't work, downgrade (max: 15 times)
- if works, mention version X as compatible version.

As you can see, the process could absolutely be scripted, thus making the process quicker.

# The time

I think this task could be worked on in parallel with the source code. However, the question of maintenance arises, but we'll leave it for now.

**Comment by HydronBlaster** [comment]
what is n

**Comment by Mansarde** [comment]
> what is n

It's just a stand-in for "any" mod.
